board/sunxi/cubie_board.c

```diff
@@ -1,26 +1,16 @@
 #include <asm-generic/gpio.h>
 #include <sunxi_gpadc.h>
 #include <dm/device.h>
+#include "cubie_board.h"
 
-#define countof(x) (sizeof(x) / sizeof(x[0]))
-#define HW_ID_ADC_CHANNEL	2
-/*PK 24 */
-#define HW_ID_GPIO			344
-
-struct hw_info_def {
-	char *compatible;
-	char *fdtfile;
-	unsigned int hw_id_level;
-	unsigned int hw_id_lower_bound;
-	unsigned int hw_id_upper_bound;
+static enum board_type board_type = BOARD_A7A;
+static const struct hw_info_def hw_info[] = {
+	{"allwinner,a733", "allwinner/sun60i-a733-cubie-a7z.dtb","PD20", "PD21", "PD22", 1, 1750, 1850}, //1800mv +/- 50mv
```

Comment:
```suggestion
	{"allwinner,a733", "allwinner/sun60i-a733-cubie-a7z.dtb", "PD20", "PD21", "PD22", 1, 1750, 1850}, //1800mv +/- 50mv
```
更好的方式是`.pcie_power_gpio = "PD20",`这样显式给field赋值

Reply:
行